extra/string-extra/StringExtension.php

```diff
@@ -79,10 +81,12 @@ public function singular(string $value, string $locale = 'en', bool $all = false
     private function getInflector(string $locale): InflectorInterface
     {
         switch ($locale) {
-            case 'fr':
-                return $this->frenchInflector ?? $this->frenchInflector = new FrenchInflector();
             case 'en':
                 return $this->englishInflector ?? $this->englishInflector = new EnglishInflector();
+            case 'es':
+                return $this->spanishInflector ?? $this->spanishInflector = new SpanishInflector();
```

Comment:
this needs to handle the case where SpanishInflector does not exist in `symfony/string`, as it does not exist for all versions supported by the requirement.

Reply:
Hi, thanks for the comment, do you any example to make the check?, and how can I test it?.

Reply:
```php
if (!class_exists(SpanishInflector::class)) {
     // throw exception with an helpful message
}
```

For testing, you would have to define 2 different tests for the 2 cases, with different `CONDITION` sections to skip the one that does not correspond to the availability of the class